opencensus/metrics/export/cumulative.py

```diff
@@ -21,18 +21,11 @@
 class CumulativePointLong(gauge.GaugePointLong):
     """A `GaugePointLong` that cannot decrease."""
 
-    def set(self, val):
-        """Set the current value to `val`
-
-        Return without changing the value if `val` is less than current value.
-
-        :type val: int
-        :param val: Value to set.
-        """
+    def _set(self, val):
         if not isinstance(val, six.integer_types):
```

Comment:
Or consider just raising an error saying `set` is not supported with cumulative?

Reply:
I'd still have to split this up since the derived point wrapper is calling `set` on both, but I may be missing you point here. In any case I think it's fine as-is, and we can refactor this if gauges and cumulatives drift further apart.

Reply:
SGTM